diesel_cli/tests/print_schema/print_schema_column_renaming/mysql/schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE with_keywords (
+    fn INTEGER NOT NULL PRIMARY KEY,
+    let INTEGER NOT NULL,
+    extern INTEGER NOT NULL
```

Comment:
Nice test table, and Imma let you merge it, but… `while self in pub do true super move`